### ISSUE TITLE
Typesafe releases fronts jCenter (and sbt marks artifact download as failed, even when artifact is downloaded successfully)

### ISSUE THREAD

**Issue by krchniam** [sbt/sbt#1821]
I stumped upon similar issue as #1820 today when upgrading to akka 2.3.9 (with sbt 0.13.7).

We have configuration, which uses our company nexus as proxy for maven central and other repositories. External resolvers are overriden, because we don't want use maven central directly, but mirrored by our proxy (we have enabled cached resolution, but that doesn't have impact on this issue):

``` scala
resolvers ++= Seq(
  Resolver.mavenLocal,
  "nexus releases" at "http://repo.company.com/nexus/content/groups/public/"
),
externalResolvers <<= resolvers map { rs =>
  Resolver.withDefaultResolvers(rs, mavenCentral = false)
}
```

Problem occured on projects where typesafe repository is present as resolver. When externalResolvers are not overriden updating works, because maven central is added before typesafe repository (works only if artifact is present in maven central). When maven central is not present, then typesafe repository is tried first and complains (`The HTTP response code for <artifact> did not indicate a success. See log for more detail.`). Then out proxy is tried and artifact downloads successfully.

```
[warn]  [FAILED     ] com.typesafe.akka#akka-actor_2.11;2.3.9!akka-actor_2.11.jar: The HTTP response code for https://repo.typesafe.com/typesafe/releases/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.jar did not indicate a success. See log for more detail. (128ms)
[info] downloading http://repo.company.com/nexus/content/groups/public/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.jar ...
[info]  [SUCCESSFUL ] com.typesafe.akka#akka-actor_2.11;2.3.9!akka-actor_2.11.jar (90ms)
```

I don't know why sbt complains `sbt.ResolveException: download failed: com.typesafe.akka#akka-actor_2.11;2.3.9!akka-actor_2.11.jar` when that artifact is dowloaded from our company proxy.

Our current workaround is restart sbt, because artifacts are dowloaded from our proxy and present in local Ivy repository (so another sbt run finds them there). According to local Ivy repository artifact is present in typesafe respository (at least pom):

```
#ivy cached data file for com.typesafe.akka#akka-actor_2.11;2.3.9
#Wed Jan 21 10:43:48 CET 2015
artifact\:akka-actor_2.11\#jar\#jar\#1899941409.exists=true
artifact\:ivy\#ivy\#xml\#575017467.location=https\://repo.typesafe.com/typesafe/releases/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.pom
artifact\:akka-actor_2.11\#pom.original\#pom\#-742463225.exists=true
resolver=sbt-chain
artifact\:ivy\#ivy\#xml\#575017467.exists=true
artifact\:akka-actor_2.11\#pom.original\#pom\#-742463225.is-local=false
artifact\:akka-actor_2.11\#pom.original\#pom\#-742463225.location=https\://repo.typesafe.com/typesafe/releases/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.pom
artifact\:akka-actor_2.11\#jar\#jar\#1899941409.is-local=false
artifact\:akka-actor_2.11\#jar\#jar\#1899941409.location=http\://repo.company.com/nexus/content/groups/public/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.jar
artifact\:ivy\#ivy\#xml\#575017467.is-local=false
```

**Comment by jsuereth** [comment]
Can you let us know which HTTP response code was returned, if not 200? I believe Ivy demands a 200.

**Comment by krchniam** [comment]
Dowloading akka-actor from https://repo.typesafe.com/typesafe/releases/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.jar returns status code 302 with header `Location: http://jcenter.bintray.com/com/typesafe/akka/akka-actor_2.11/2.3.9/akka-actor_2.11-2.3.9.jar`

**Comment by eed3si9n** [comment]
On Friday, repo.typesafe.com stopped fronting jCenter, so this problem should be solved for akka-actor 2.3.9.

But the underlying problem remains for the artifacts like Slick Extensions that _are_ hosted by Typesafe repo which _does_ return HTTP 302 Found redirect to Bintray. Is this ultimately an issue with Ivy and/or Nexus?
- [NEXUS-5745: Detect proxy redirects and do something with it (Won't Fix)](https://issues.sonatype.org/browse/NEXUS-5745)

**Comment by eed3si9n** [comment]
Given the same error message, I am closing this as a duplicate of #1820.
